webpack.playground.config.js

```diff
@@ -9,6 +10,14 @@ const PLAYGROUND = path.resolve(__dirname, 'playground');
 const DEFAULT_SERVER_PORT = 3000;
 const WIDGET_RC_FILE = '.widgetrc.js';
 
+if (!fs.existsSync(WIDGET_RC_FILE) && fs.existsSync('.widgetrc')) {
+  console.log('============================================');
+  console.log('Please migrate the .widgetrc to .widgetrc.js.');
+  console.log('Check details at https://github.com/okta/okta-signin-widget/blob/master/MIGRATING.md');
+  console.log('============================================');
```

Comment:
Can we make this message more apparent we're terminating the process by using `console.error`?

Reply:
I don't see difference at console but I'll change anyway.